ui/raidboss/emulator/data/CombatantJobSearch.ts

```diff
@@ -1,14 +1,16 @@
+import { Job } from '../../../../types/job';
+
 export default class CombatantJobSearch {
-  static getJob(abilityId: number): string | undefined {
-    for (const job in CombatantJobSearch.abilities) {
-      if (CombatantJobSearch.abilities[job]?.includes(abilityId))
-        return job;
+  static getJob(abilityId: number): Job | undefined {
+    for (const [key, value] of Object.entries(CombatantJobSearch.abilities)) {
+      if (value?.includes(abilityId))
+        return key as Job;
```

Comment:
```suggestion
        return key as keyof typeof CombatantJobSearch.abilities;
```

I realize it's like two lines below, but at least this validates that if you changed the type of `abilities` you would also have to fix this up to be correct as well.